lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -81,12 +83,19 @@ class ScreenshotThumbnails extends Audit {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
     const cachedThumbnails = new Map();
 
-    return artifacts.requestSpeedline(trace).then(speedline => {
+    return Promise.all([
+      artifacts.requestSpeedline(trace),
+      TTFI.audit(artifacts).catch(() => ({})),
```

Comment:
yes, but the consistency in testing and referencing `rawValue` seemed to provide some value whereas I'm not really sure what we gain by using the computed artifact just for one of them here